### PR TITLE
Add backoffice filter from old ralph

### PR DESCRIPTION
**Pull request by szok**
- Added missing filter to backoffice hardware
- Added support foreign key field in filter
- Added AssetHolder model
- Added property of field to BackOfficeAsset

### CODE REVIEW

File: src/ralph/assets/models/assets.py [hunk 1]
```diff
@@ -150,7 +150,7 @@ class Meta:
         verbose_name_plural = _('models')
 
     def __str__(self):
-        if self.category:
+        if self.category_id:
```
Comment (mkurek) [inline src/ralph/assets/models/assets.py:153]: :+1:

File: src/ralph/back_office/models.py [hunk 2]
```diff
@@ -27,6 +27,10 @@ class Warehouse(NamedMixin, TimeStampMixin, models.Model):
     pass
 
 
+class AssetOwner(NamedMixin.NonUnique, TimeStampMixin, models.Model):
```
Comment (mkurek) [inline src/ralph/back_office/models.py]: it is confusing with user/owner of the asset - IMO we should rename it. @vi4m any idea how to name it?

**Comment by ar4s** [comment]
:ok:

File: src/ralph/admin/filters.py [hunk 3]
```diff
@@ -205,14 +209,62 @@ def choices(self, cl):
...
 
-    @property
-    def choices_list(self):
-        model = get_model_from_relation(self.field)
-        return [(i.id, str(i)) for i in model._default_manager.all()]
+    template = "admin/filters/related_filter.html"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.model_field = get_model_from_relation(self.field)
```
Comment (mkurek) [inline src/ralph/admin/filters.py]: wouldn't `field_model` be better here?

**Comment by mkurek** [comment]
:ok:

**Pull request merged**